src/base/emu-i386/simx86/trees.c

```diff
@@ -970,6 +971,8 @@ TNode *Move2Tree(IMeta *I0, CodeBuf *GenCodeBuf)
 	}
 	/* ->REPLACE the code of the node found with the latest
 	   compiled version */
+	if (ng->alive > 0)
```

Comment:
So is this a typo you fix in a
subsequent patch??

Reply:
Must have done a one off mistake during the interactive rebase... Will fix!

Reply:
done